### PR TITLE
fixes of i18n on get-an-account-orderform-configuration.md

### PR DESCRIPTION
**Pull request by GeorgeLimaDev**
Fixget-an-account-orderform-configuration.md
It relates to [LOC-10431](https://vtex-dev.atlassian.net/browse/LOC-10431) and KR 23Q2-KR1 ([Review all developer portal documentation in English](https://docs.google.com/document/d/1VduAYpjqdazhyGH5DvD9c52pMu1Xfj4RDx48QNjTFU0/edit))

#### Types of changes
- [ ] New content (guides, endpoints, app documentation)
- [ ] Improvement (make a documentation even better)
- [ ] Fix (fix a documentation error)
- [X] Spelling and grammar accuracy (self-explanatory)

[LOC-10431]: https://vtex-dev.atlassian.net/browse/LOC-10431?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-an-account-orderform-configuration.md [hunk 1]
```diff
@@ -1,46 +1,38 @@
...
-         - `paymentSystemToCheckFirstInstallment`: option to apply a first installment discount to a particular payment system.
+This guide describes how to check the following settings, which are currently applied to every orderForm in a specific account:
 
-- `taxConfiguration`: external tax service configuration information.
-         - `url`: endpoint URL.
-         - `authorizationHeader`: authorization header.
-         - `appId`: custom data ID sent to the tax system.
+- `paymentConfiguration`: Payment configuration information. -  `requiresAuthenticationForPreAuthorizedPaymentOption`: Determines whether pre-authorized payments require authentication. - `allowInstallmentsMerge`: In a multi-seller purchase scenario, it allows a flexible installment option that considers maximum installments for each seller according to their respective configuration options. - `paymentSystemToCheckFirstInstallment`: Option to apply a first installment discount to a particular payment system.
```
Comment (carolinamenezes) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-an-account-orderform-configuration.md:13]:
```suggestion
- `paymentConfiguration`: Payment configuration information. 
     - `requiresAuthenticationForPreAuthorizedPaymentOption`: Determines whether pre-authorized payments require authentication. 
     - `allowInstallmentsMerge`: In a multi-seller purchase scenario, it allows a flexible installment option that considers maximum installments for each seller according to their respective configuration options. 
     - `paymentSystemToCheckFirstInstallment`: Option to apply a first installment discount to a particular payment system.
```

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-an-account-orderform-configuration.md [hunk 2]
```diff
@@ -1,46 +1,38 @@
...
 
-- `minimumQuantityAccumulatedForItems`: minimum SKU quantity by cart.
-- `decimalDigitsPrecision`: number of price digits.
-- `minimumValueAccumulated`: minimum cart value.
-- `apps`: apps configuration information.
-         - `id`: app ID.
-         - `fields`: app fields information.
-         - `major`: app major version.
+- `taxConfiguration`: External tax service configuration information. - `url`: Endpoint URL. - `authorizationHeader`: Authorization header. - `appId`: Custom data ID sent to the tax system.
```
Comment (carolinamenezes) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-an-account-orderform-configuration.md:15]:
```suggestion
- `taxConfiguration`: External tax service configuration information. 
     - `url`: Endpoint URL. 
     - `authorizationHeader`: Authorization header. 
     - `appId`: Custom data ID sent to the tax system.
```

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-an-account-orderform-configuration.md [hunk 3]
```diff
@@ -1,46 +1,38 @@
...
-- `allowMultipleDeliveries`: allows the selection of items from several delivery channels in the same purchase.
-- `allowManualPrice`: allows the editing of SKU prices directly in the cart.
-- `savePersonalDataAsOptIn`: allows users to select whether they want the store to keep their personal and payment data saved.
-- `maxNumberOfWhiteLabelSellers`: allows the input of a limit of white label sellers involved on the cart.
-- `maskFirstPurchaseData`: allows, on a first purchase, masking client's data. It could be useful when a shared cart is used and the client doesn't want to share its data.
-- `recaptchaValidation`: configures reCAPTCHA validation status for the account.
+- `minimumQuantityAccumulatedForItems`: Minimum SKU quantity per cart.
+- `decimalDigitsPrecision`: Number of price digits.
+- `minimumValueAccumulated`: Minimum cart value.
+- `apps`: Apps configuration information. - `id`: App ID. - `fields`: App fields information. - `major`: App major version.
```
Comment (carolinamenezes) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-an-account-orderform-configuration.md:20]:
```suggestion
- `apps`: Apps configuration information. 
     - `id`: App ID. 
     - `fields`: App fields information. 
     - `major`: App major version.
```

File: docs/guides/Checkout/configuration-section-api-quick-start-guides/get-an-account-orderform-configuration.md [hunk 4]
```diff
@@ -127,14 +120,13 @@ The following errors may appear as a message in the response body.
 }
 ```
 
-
 ### 404 - Not Found
 
-- **Message error example**: `"The requested URL was not found on the server"`: check that the URL data is correct.
+- **Message error example**: `"The requested URL was not found on the server"`. Check if the URL is correct.
 
 ```json
```
Comment (carolinamenezes) [inline docs/guides/Checkout/configuration-section-api-quick-start-guides/get-an-account-orderform-configuration.md:127]:
```suggestion
```html
```

**Review by carolinamenezes — COMMENTED** [review]
Algumas listas estão desformatadas

**Review by karenkrieger — APPROVED** [review]
LGTM

**Pull request merged**